frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -77,6 +80,7 @@ function pickRowsToMeasure(rows, columnIndex, count = 10) {
 
 const mapStateToProps = state => ({
   queryBuilderMode: getQueryBuilderMode(state),
+  PKRowIndexMap: getPKRowIndexMap(state),
```

Comment:
There's a very similar bug: #34070
Unfortunately this PR does not seem fix it.

Reply:
seems it's a different problem, which requires more investigation

Reply:
fa0289f5644f3b912a901a4bdafc58f65468e797 should fix it